models/hl7-v3/dispense-claim.ts

```diff
@@ -78,6 +79,7 @@ export class DispenseClaimSupplyHeader {
   code: codes.SnomedCode
   effectiveTime: core.Null
   repeatNumber?: core.Interval<core.NumericValue>
+  author: Author
```

Comment:
DispenseClaim doesn't have an `author` field, it has a `legalAuthenticator` field.